pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -1002,12 +1002,25 @@ def make_backup(self):
             logging.warn(f'{backup_dir} already exists')
             self.set_state(TestState.ERROR)
             return
+        # Make a RockDB snapshot of the db to save space. This takes advantage of the immutability of sst files.
         logging.info(f'copying data dir to {backup_dir}')
-        shutil.copytree(self.target_near_home_path('data'),
-                        backup_dir,
-                        dirs_exist_ok=True)
-        logging.info(f'copied data dir to {backup_dir}')
-
+        cmd = [
+            self.data['current_neard_path'], '--home',
+            self.target_near_home_path(),
+            '--unsafe-fast-startup',
+            'database',
+            'make-snapshot',
+            '--destination',
+            backup_dir
+        ]
+        self.run_neard(cmd)
+        running = True
+        while running:
+            _, running, exit_code = self.poll_neard()
+        logging.info(f'copying data dir to {backup_dir} finished with code {exit_code}')
+        # Copy config, genesis and node_key to the backup folder to use them to restore the db.
+        for file in ["config.json", "genesis.json", "node_key.json"]:
+            shutil.copyfile(self.target_near_home_path(file), os.path.join(backup_dir, file))
```

Comment:
Why do you need to copy those? I thought that it's only that data directory that needs backup. 

I think there are a few more files in the the home directory like validator key, records or log_config what about those?

Reply:
Given:
`~/.near` the home directory of the node.
`~/backup/x` the path to the backup folder `x`, this will contain the data folder: `~/backup/x/data`

I proposed to restore the data partition by running `neard --home ~/backup/x database make_backup --destination ~/.near`. This will open the backup x and save it to the home folder of a node.